docs/custom_events.md

```diff
@@ -0,0 +1,50 @@
+# Using CloudWatch RUM Web Client to Record Custom Events
```

Comment:
Thought a dedicated document would be better than appending onto `docs/cdn_commands.md`. Let me know if you think otherwise.